websites/P/Peloton/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+        "name": "The Gamerzs",
+        "id": "712294190339588209"
+    },
+    "url": [
```

Comment:
Single URL, please use string instead of an array.

Reply:
Added other urls